tests/Serializer/GraphNavigatorTest.php

```diff
@@ -128,17 +129,42 @@ public function testNavigatorChangeTypeOnSerialization()
 
         $navigator = new SerializationGraphNavigator($this->metadataFactory, $this->handlerRegistry, $this->accessor, $this->dispatcher);
         $navigator->initialize($this->serializationVisitor, $this->context);
+        $this->context->initialize(TestSubscribingHandler::FORMAT, $this->serializationVisitor, $navigator, $this->metadataFactory);
```

Comment:
This is added here because the `$this-context` mock is changed and initialization now needs to happen.
Also see https://github.com/schmittjoh/serializer/pull/1169#discussion_r391545324